tests/vm_test.go

```diff
@@ -191,7 +190,7 @@ var _ = Describe("[rfe_id:1177][crit:medium][vendor:cnv-qe@redhat.com][level:com
 				libvmi.WithDataVolume("disk0", dataVolume.Name),
 				libvmi.WithResourceMemory("256Mi"),
 				libvmi.WithNamespace(testsuite.GetTestNamespace(nil)),
-				libvmi.WithCloudInitNoCloud(libvmici.WithNoCloudEncodedUserData("#!/bin/bash\necho hello\n")),
+				libvmi.WithCloudInitNoCloud(libvmifact.WithEchoHelloUserData()),
```

Comment:
how about libvmifact.WithQuickerBootNoCloud() ?

Reply:
WithDummyCloudInitForQuickerBoot?

Reply:
Changed to WithDummyCloudForFastBoot